packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -1839,3 +1839,158 @@ export function openClose(componentTagOrHTML: TagOrHTML, options?: OpenCloseOpti
+    rgb(232, 192, 83),
+    rgb(237, 189, 106),
+    rgb(244, 147, 90),
+    rgb(204, 121, 32),
+    rgb(216, 131, 88),
+    rgb(221, 148, 93),
+    rgb(255, 144, 96),
+    rgb(242, 124, 33),
+    rgb(196, 69, 5),
+    rgb(237, 137, 87),
+    rgb(229, 194, 41),
+    rgb(252, 244, 95),
+    rgb(241, 244, 36),
+    rgb(216, 189, 54),
+    rgb(229, 218, 64),
+    rgb(252, 235, 106),
+    rgb(226, 220, 102),
+    rgb(234, 227, 98),
+    rgb(244, 244, 4),
+    rgb(229, 212, 100),
+    rgb(226, 216, 63),
+    rgb(247, 214, 81),
+    rgb(247, 232, 64),
+    rgb(224, 195, 80),
+    rgb(242, 230, 106),
+    rgb(13, 232, 199),
+    rgb(102, 160, 9),
+    rgb(4, 158, 45),
+    rgb(29, 193, 97),
+    rgb(6, 232, 127),
+    rgb(88, 181, 30),
+    rgb(115, 175, 31),
+    rgb(43, 229, 114),
+    rgb(106, 252, 95),
+    rgb(66, 255, 166),
+    rgb(101, 221, 95),
+    rgb(114, 255, 240),
+    rgb(188, 221, 88),
+    rgb(185, 219, 15),
+    rgb(64, 209, 187),
+    rgb(76, 119, 173),
+    rgb(74, 124, 181),
+    rgb(4, 54, 204),
+    rgb(11, 170, 188),
+    rgb(128, 98, 219),
+    rgb(3, 135, 150),
+    rgb(92, 214, 212),
+    rgb(46, 136, 232),
+    rgb(86, 70, 168),
+    rgb(32, 75, 173),
+    rgb(28, 131, 165),
+    rgb(99, 99, 221),
+    rgb(105, 177, 244),
+    rgb(27, 112, 119),
+    rgb(78, 197, 252),
+    rgb(57, 10, 168),
+    rgb(172, 54, 226),
+    rgb(152, 75, 252),
+    rgb(158, 15, 224),
+    rgb(117, 0, 196),
+    rgb(56, 10, 119),
+    rgb(139, 90, 237),
+    rgb(116, 65, 198),
+    rgb(90, 11, 130),
+    rgb(98, 18, 135),
+    rgb(135, 38, 181),
+    rgb(113, 61, 211),
+    rgb(163, 29, 247),
+    rgb(74, 27, 145),
+    rgb(128, 79, 188),
+    rgb(216, 0, 255),
+    rgb(221, 88, 175),
+    rgb(249, 29, 187),
+    rgb(244, 78, 172),
+    rgb(242, 77, 168),
+    rgb(249, 49, 136),
+    rgb(239, 57, 239),
+    rgb(234, 42, 212),
+    rgb(242, 94, 215),
+    rgb(211, 74, 177),
+    rgb(206, 24, 219),
+    rgb(207, 41, 244),
+    rgb(242, 107, 249),
+    rgb(226, 6, 190),
+    rgb(214, 40, 237),
+  ];
+
+  function shuffle(array) {
+    let currentIndex = array.length;
+    let temporaryValue;
+    let randomIndex;
+
+    // While there remain elements to shuffle...
+    while (0 !== currentIndex) {
+      // Pick a remaining element...
+      randomIndex = Math.floor(Math.random() * currentIndex);
+      currentIndex -= 1;
+
+      // And swap it with the current element.
+      temporaryValue = array[currentIndex];
+      array[currentIndex] = array[randomIndex];
+      array[randomIndex] = temporaryValue;
+    }
+
+    return array;
+  }
+
+  colorList = shuffle(colorList);
+
+  const page = await simplePageSetup(componentTagOrHTML);
+  const tag = getTag(isHTML(componentTagOrHTML) ? componentName : componentTagOrHTML);
+  const component = await page.find(tag);
+  const themedTokens = {};
+
+  tokens.forEach((token, i) => {
+    themedTokens[token] = token.includes("color") ? colorList[i] : `${i * 10}${token.includes("z-index") ? "" : "px"}`;
```

Comment:
nice. 👍 

This might not cover everything though, like shadows, and stuff. I wonder if we should have some kind of lookup function to map all these things? I think we can search for things like space, color, z-index, shadow, anything else?

Reply:
updated to cover those use cases